.mailmap

```diff
@@ -0,0 +1,12 @@
+# Prevent git from showing duplicate names with commands like "git shortlog"
+# See the manpage of git-shortlog for details.
+# The syntax is:
+# Name that should be used <email that should be used> Bad name <bad email>
+#
+# You can skip Bad name if it is the same as the one that should be used, and is unique.
+#
+# This file is up-to-date if the command git log --format="%aN <%aE>" | sort -u
+# gives no duplicates.
```

Comment:
This gives:
```
Alan Bromborsky <abrombo@users.noreply.github.com>
alex arsenovic <REDACTED1@gmail.com>
arsenovic <REDACTED1@gmail.com>
Eric Wieser <REDACTED2@gmail.com>
eric-wieser <REDACTED2@gmail.com>
hugo hadfield <REDACTED3@gmail.com>
Hugo Hadfield <REDACTED3@gmail.com>
hugohadfield <REDACTED3@gmail.com>
Sylvain Meunier <REDACTED4@gmail.com>
utensil <REDACTED5@gmail.com>
Utensil Song <REDACTED5@gmail.com>
Waldir Pimenta <REDACTED6@gmail.com>
```

Still duplicates, but the only way to fix them is to put your email address in a more visible place than before, which is something I should let people opt into individually.